fixture/vcr_cassettes/giphy_search_steve.json

```diff
@@ -0,0 +1,35 @@
":\"433880\",\"mp4\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/200w.mp4\",\"mp4_size\":\"10866\",\"webp\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/200w.webp\",\"webp_size\":\"185952\"},\"fixed_width_still\":{\"url\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/200w_s.gif\",\"width\":\"200\",\"height\":\"111\"},\"fixed_width_downsampled\":{\"url\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/200w_d.gif\",\"width\":\"200\",\"height\":\"111\",\"size\":\"96673\",\"webp\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/200w_d.webp\",\"webp_size\":\"25220\"},\"fixed_height_small\":{\"url\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/100.gif\",\"width\":\"181\",\"height\":\"100\",\"size\":\"395013\",\"mp4\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/100.mp4\",\"mp4_size\":\"10177\",\"webp\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/100.webp\",\"webp_size\":\"168978\"},\"fixed_height_small_still\":{\"url\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/100_s.gif\",\"width\":\"181\",\"height\":\"100\"},\"fixed_width_small\":{\"url\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/100w.gif\",\"width\":\"100\",\"height\":\"55\",\"size\":\"117479\",\"mp4\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/100w.mp4\",\"mp4_size\":\"5261\",\"webp\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/100w.webp\",\"webp_size\":\"73008\"},\"fixed_width_small_still\":{\"url\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/100w_s.gif\",\"width\":\"100\",\"height\":\"55\"},\"downsized\":{\"url\":\"http:\\/\\/media.giphy.com\\/media\\/11vzuRDLBNuuAg\\/giphy-tumblr.gif\",\"width\":\"250\",\"height\":\"138\",\"size\":\"738567\"},\"downsized_still\":{\"url\":\"http:\\/\\/media.giphy.com\\/media\\/11vzuRDLBNuuAg\\/giphy-tumblr_s.gif\",\"width\":\"250\",\"height\":\"138\"},\"downsized_large\":{\"url\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/giphy.gif\",\"width\":\"608\",\"height\":\"336\",\"size\":\"4029186\"},\"downsized_medium\":{\"url\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/giphy.gif\",\"width\":\"608\",\"height\":\"336\",\"size\":\"4029186\"},\"original\":{\"url\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/giphy.gif\",\"width\":\"608\",\"height\":\"336\",\"size\":\"4029186\",\"frames\":\"45\",\"mp4\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/giphy.mp4\",\"mp4_size\":\"36429\",\"webp\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/giphy.webp\",\"webp_size\":\"810106\"},\"original_still\":{\"url\":\"http:\\/\\/media1.giphy.com\\/media\\/11vzuRDLBNuuAg\\/giphy_s.gif\",\"width\":\"608\",\"height\":\"336\"},\"looping\":{\"mp4\":\"http:\\/\\/media.giphy.com\\/media\\/11vzuRDLBNuuAg\\/giphy-loop.mp4\"}}}],\"meta\":{\"status\":200,\"msg\":\"OK\",\"response_id\":\"57f117bf8a100b764c87df7c\"},\"pagination\":{\"total_count\":12327,\"count\":10,\"offset\":0}}",
```

Comment:
It's auto-generated by `ex_vcr` so, yes *until* we need to change it, then we'll have to break it up again. To me, feels like "not for humans"... It may become a non-issue when I build my own mock object.